app/services/sqs/message_publisher.rb

```diff
@@ -21,6 +22,14 @@ def messaging_enabled?
       queue_url.present?
     end
 
-    attr_reader :sqs_client, :message, :queue_url
+    def log_message
+      log_info.merge(
+        request_id: Current.request_id,
+        queue: queue_url,
+      ).map { |k, v| "#{k}: #{v}" }
+       .join(", ")
```

Comment:
It takes the hash `log_info` and merges additional key-value pairs into it. It then transforms the result into a string representation of each pair and then joins all the strings together with commas.  The resulting string is used for the logging.